lib/morph/docker_runner.rb

```diff
@@ -41,21 +41,14 @@ def self.compile_and_start_run(
       # If something went wrong during the compile and it couldn't finish
       return [nil, nil] if i3.nil?
 
-      # Insert the actual code (and database) into the container
-      i4 = Dir.mktmpdir('morph') do |dest|
-        copy_config_to_directory(repo_path, dest, false)
-        yield(:internalout, "Injecting scraper and running...\n")
-        inject_files2(i3, dest)
```

Comment:
I don't think the `.inject_files2` method is used after this change so it can be removed.